### DISCUSSION TITLE
Questions about MIMIC-IV ECHO and MIMIC-IV ECG.

### DISCUSSION THREAD

**Discussion by danielemolino** [MIT-LCP/mimic-code#1761] [MIMIC-IV]
Hi everyone,

First of all, I'd like to thank the team for the brilliant work on this project. I really appreciate the effort and dedication that has gone into developing and maintaining this dataset. I should mention that my questions might seem a bit basic as I don't have advanced medical knowledge, so I hope you can bear with me.

I have two questions:

1) When I open the DICOM files related to echocardiograms with a viewer software, I notice that apart from the images, there's a signal related to the heart's activity along with a pointer indicating the correlation between the frames and the signal. However, I haven't been able to determine which DICOM field contains this information. Could someone please point me in the right direction?

2) Is there any correlation between the ECG studies of the MIMIC-IV ECG and the echocardiograms in MIMIC-IV ECHO? Specifically, is it possible to find a link between these two types of data?

Thank you in advance for your help!

**Comment by briangow** [comment]
@danielemolino , sorry for the late response. 

> When I open the DICOM files related to echocardiograms with a viewer software, I notice that apart from the images, there's a signal related to the heart's activity along with a pointer indicating the correlation between the frames and the signal. However, I haven't been able to determine which DICOM field contains this information. Could someone please point me in the right direction?

Unfortunately, the sample information for the heart activity signal that appears on top of some DICOMs is not available in the metadata. 

> Is there any correlation between the ECG studies of the MIMIC-IV ECG and the echocardiograms in MIMIC-IV ECHO? Specifically, is it possible to find a link between these two types of data?

There is no direct correlation. However, datetime shifts in MIMIC are consistent for a given patient across all of the modules. Therefore, you can use datetime to align the ECGs & Echos for a given patient.
